src/main/java/mate/academy/spring/validation/Password.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+
+@Constraint(validatedBy = PasswordValidator.class)
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Password {
+    String message() default "Password and repeat password values don't match!";
+
+    String password();
+
+    String repeatPassword();
+
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        Password[] value();
+    }
```

Comment:
what for?

Reply:
`We can see our custom annotation also contains a List sub-interface for defining multiple FieldsValueMatch annotations on a class.`

[We can see our custom annotation also contains a List sub-interface for defining multiple FieldsValueMatch annotations on a class.](https://www.baeldung.com/spring-mvc-custom-validator#1-creating-the-annotation)